src/components/Explore/BalanceSummary.tsx

```diff
@@ -1,8 +1,9 @@
+import { TEXT_COLORS } from 'components/NetworkAlert/NetworkAlert'
```

Comment:
put this in a shared file

Reply:
because it's not in the same component file, where do you suggest i put this? (ie this isn't in components/Explore)

Reply:
I guess I could just put a floater file in components but it doesn't seem like that's best practice? thoughts?